libs/atqr/domain/src/lib/domain-spec/stories/playerReceivesCondolences.disl

```diff
@@ -0,0 +1,13 @@
+read_also api.disl
+read_also enums.disl
+read_also ui.disl
+
+doman/NotificationService calls ChallengeFailed!
+    rules:
+        - JudgementStatus == Judged
+        - ChallengeStatus == Finished
+        - PlayerOutcome == Failed
+
+    when. 
+        Challenge.PlayerOutcome is Failed
+            NotificationService will send ChallengeFailed! email
```

Comment:
You just need to fire the `ChallengeFailed!` event. Also, you cannot tell another class to fire an event